website/layouts/shortcodes/openssl-version.html

```diff
@@ -0,0 +1,3 @@
+{{ $package := partialCached "cargo-package" "openssl-src" }}
+{{ $openssl_version := index (strings.Split $package.version "+") 1 }}
+{{ with .Get "majMin" }}{{ with strings.Split $openssl_version "." }}{{ index . 0 }}.{{ index . 1 }}{{ end }}{{ else }}{{ $openssl_version }}{{ end }}
```

Comment:
```suggestion
{{- $package := partialCached "cargo-package" "openssl-src" -}}
{{- $openssl_version := index (strings.Split $package.version "+") 1 -}}
{{- with .Get "majMin" -}}{{- with strings.Split $openssl_version "." -}}{{- index . 0 -}}.{{- index . 1 -}}{{- end -}}{{- else -}}{{- $openssl_version -}}{{- end -}}
```

Hugo does a great job in situations like this, but it can be tricky sometimes with how things render (and render unexpectedly, I've fallen victim to this a few times now  😅). By adding a `-` directly after the `{{` and directly before `}}` we ensure no unintended whitespace and new lines are created when this renders to the page at build time.

Reply:
Ah I was wondering what was the difference in these tags. I couldn't find very clear documentation about them. Learned something new! And agree with your change accordingly.  👍

Reply:
Oh! These are golang text template tags ! Now I realized and found the documentation about it :facepalm: